nemo/collections/nlp/models/language_modeling/megatron_lm_encoder_decoder_model.py

```diff
@@ -407,8 +407,9 @@ def predict_step(self, batch: Any, batch_idx: int, dataloader_idx: Optional[int]
         logging.info(f"response: {response}")
         return response
 
-    def decode(self, tokens_enc, enc_mask, num_tokens_to_generate, enc_input=None):
+    def decode(self, tokens_enc, enc_mask, num_tokens_to_generate, enc_input=None, tokenizer=None):
```

Comment:
Why is tokenizer can be given as an argument?

Reply:
This is because in T5 style models, we could get away with always accessing `self.tokenizer` to do the IDs -> text conversion, but for NMT, we now have `self.encoder_tokenizer` and `self.decoder_tokenizer`.